sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/SqlStdOperatorMappingTable.java

```diff
@@ -0,0 +1,359 @@
+ */
+package org.apache.beam.sdk.extensions.sql.zetasql;
+
+import com.google.zetasql.ZetaSQLFunction.FunctionSignatureId;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.beam.sdk.annotations.Internal;
+import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.collect.ImmutableMap;
+import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.collect.ImmutableSet;
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+
+/** SqlStdOperatorMappingTable. */
+@Internal
+public class SqlStdOperatorMappingTable {
+  static final List<FunctionSignatureId> ZETASQL_BUILTIN_FUNCTION_WHITELIST =
+      Arrays.asList(
+          FunctionSignatureId.FN_AND,
+          FunctionSignatureId.FN_OR,
+          FunctionSignatureId.FN_NOT,
+          FunctionSignatureId.FN_MULTIPLY_DOUBLE,
+          FunctionSignatureId.FN_MULTIPLY_INT64,
+          FunctionSignatureId.FN_MULTIPLY_NUMERIC,
+          FunctionSignatureId.FN_DIVIDE_DOUBLE,
+          FunctionSignatureId.FN_DIVIDE_NUMERIC,
+          FunctionSignatureId.FN_ADD_DOUBLE,
+          FunctionSignatureId.FN_ADD_INT64,
+          FunctionSignatureId.FN_ADD_NUMERIC,
+          FunctionSignatureId.FN_SUBTRACT_DOUBLE,
+          FunctionSignatureId.FN_SUBTRACT_INT64,
+          FunctionSignatureId.FN_SUBTRACT_NUMERIC,
+          FunctionSignatureId.FN_UNARY_MINUS_INT64,
+          FunctionSignatureId.FN_UNARY_MINUS_DOUBLE,
+          FunctionSignatureId.FN_UNARY_MINUS_NUMERIC,
+          FunctionSignatureId.FN_GREATER,
+          FunctionSignatureId.FN_GREATER_OR_EQUAL,
+          FunctionSignatureId.FN_LESS,
+          FunctionSignatureId.FN_LESS_OR_EQUAL,
+          FunctionSignatureId.FN_EQUAL,
+          FunctionSignatureId.FN_NOT_EQUAL,
+          FunctionSignatureId.FN_IS_NULL,
+          FunctionSignatureId.FN_IS_TRUE,
+          FunctionSignatureId.FN_IS_FALSE,
+          FunctionSignatureId.FN_STARTS_WITH_STRING,
+          FunctionSignatureId.FN_SUBSTR_STRING,
+          FunctionSignatureId.FN_TRIM_STRING,
+          FunctionSignatureId.FN_LTRIM_STRING,
+          FunctionSignatureId.FN_RTRIM_STRING,
+          FunctionSignatureId.FN_REPLACE_STRING,
+          FunctionSignatureId.FN_CONCAT_STRING,
+          FunctionSignatureId.FN_COUNT_STAR,
+          FunctionSignatureId.FN_COUNT,
+          FunctionSignatureId.FN_MAX,
+          FunctionSignatureId.FN_MIN,
+          FunctionSignatureId.FN_AVG_DOUBLE,
+          FunctionSignatureId.FN_AVG_INT64,
+          FunctionSignatureId.FN_AVG_NUMERIC,
+          FunctionSignatureId.FN_SUM_DOUBLE,
+          FunctionSignatureId.FN_SUM_INT64,
+          FunctionSignatureId.FN_SUM_NUMERIC,
+          FunctionSignatureId.FN_MOD_INT64,
+          FunctionSignatureId.FN_MOD_NUMERIC,
+          FunctionSignatureId.FN_CASE_NO_VALUE,
+          FunctionSignatureId.FN_CASE_WITH_VALUE,
+          FunctionSignatureId.FN_TIMESTAMP_ADD,
+          // FunctionSignatureId.FN_TIMESTAMP_SUB,
```

Comment:
There are a whole bunch of these commented out values, should they just be deleted? (I'm guessing they are effectively TODOs, but want to confirm.)

Reply:
They are effectively TODO or partially supported. I will add comment to mark it.